client/platform/web-girder/views/Admin/AdminJobs.vue

```diff
@@ -0,0 +1,437 @@
+              <div v-if="item.params.dataset_input_list">
+                <v-tooltip
+                  bottom
+                >
+                  <template #activator="{on, attrs}">
+                    <v-btn
+                      v-bind="attrs"
+                      x-small
+                      depressed
+                      color="info"
+                      class="ml-0"
+                      v-on="on"
+                      @click="viewTrainingList(item.params.dataset_input_list)"
+                    >
+                      <v-icon small>
+                        mdi-eye
+                      </v-icon>
+                    </v-btn>
+                  </template>
+                  <span>View Training List</span>
+                </v-tooltip>
+              </div>
+            </div>
+          </template>
+          <template v-slot:item.actions="{ item }">
+            <v-tooltip bottom>
+              <template #activator="{on, attrs}">
+                <v-btn
+                  v-bind="attrs"
+                  x-small
+                  depressed
+                  :href="`/girder/#job/${item.actions}`"
+                  color="info"
+                  class="mx-2 my-2"
+                  v-on="on"
+                >
+                  <v-icon small>
+                    mdi-text-box-outline
+                  </v-icon>
+                </v-btn>
+              </template>
+              <span>View job logs and manage job</span>
+            </v-tooltip>
+            <v-tooltip
+              v-if="item.status < 3 "
+              bottom
+            >
+              <template #activator="{on, attrs}">
+                <v-btn
+                  v-bind="attrs"
+                  x-small
+                  depressed
+                  color="warning"
+                  class="mx-2 my-2"
+                  v-on="on"
+                  @click="modifyJob('Cancel', item.actions, item.title)"
+                >
+                  <v-icon small>
+                    mdi-close-circle-outline
+                  </v-icon>
+                </v-btn>
+              </template>
+              <span>Cancel Job</span>
+            </v-tooltip>
+            <v-tooltip
+              bottom
+            >
+              <template #activator="{on, attrs}">
+                <v-btn
+                  v-bind="attrs"
+                  x-small
+                  depressed
+                  color="error"
+                  class="mx-2 my-2"
+                  v-on="on"
+                  @click="modifyJob('Delete', item.actions, item.title)"
+                >
+                  <v-icon small>
+                    mdi-delete
+                  </v-icon>
+                </v-btn>
+              </template>
+              <span>Delete Job</span>
+            </v-tooltip>
+          </template>
+        </v-data-table>
+      </v-card-text>
+    </v-card>
+    <v-dialog
+      v-model="trainingListDialog"
+      width="250"
+    >
+      <v-card>
```

Comment:
The dialog blends into the background too much for me. Can we add `color="blue-grey darken-4"` for better contrast?